d2game/d2player/mini_panel.go

```diff
@@ -262,9 +262,7 @@ func (m *miniPanel) Toggle() {
 }
 
 func (m *miniPanel) Open() {
-	if !m.movedLeft && !m.movedRight {
-		m.panelGroup.SetVisible(true)
-	}
+	m.panelGroup.SetVisible(true)
```

Comment:
I don't think this is the right solution. If the both the left and the right panel are open, we should not be able open the minipanel.

Reply:
thanks @juander-ux. Now should be OK

Reply:
From retail d2:
![ezgif-3-971b7e263fe4](https://user-images.githubusercontent.com/936440/100395666-9ac51680-2ff6-11eb-9377-d384b427b94c.gif)